saleor/checkout/utils.py

```diff
@@ -969,3 +969,10 @@ def get_or_create_checkout_metadata(checkout: "Checkout"):
         return checkout.metadata_storage
     else:
         return CheckoutMetadata.objects.create(checkout=checkout)
+
+
+def get_checkout_metadata(checkout: "Checkout"):
```

Comment:
Can we add unit tests for this function? They should check both scenarios - saving metadata to a checkout where storage already exists and to a checkout without it.

Reply:
Done